py/mpconfig.h

```diff
@@ -850,6 +850,19 @@ typedef double mp_float_t;
 #define MICROPY_MODULE_BUILTIN_INIT (MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES)
 #endif
 
+// Whether to allow built-in modules to have sub-packages (by making the
+// sub-package a member of their locals dict). Sub-packages should not be
+// registered with MP_REGISTER_MODULE, instead they should be added as
+// members of the parent's globals dict. Their __name__ should be foo.bar
+// (i.e. QSTR_foo_dot_bar) which will require an entry in qstrdefs. Also,
```

Comment:
The git commit message says that `__name__` could be `bar` as well.  Is that correct?

Reply:
Updated.

Reply:
thanks